_docs/reference/api/attribute-vocabulary.md

```diff
@@ -39,16 +39,16 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | target.labels | map | A map of key-value pairs attached to the target. | |
 | target.user | string | The user running the target application. | service-account |
 | request.headers | map | A map of HTTP headers attached to the request. | |
-| request.id | string | A unique ID for the request, which can be propagated to downstream systems. This should be a guid or a psuedo-guid with a low probability of collision in a temporal window measured in days or weeks. | |
+| request.id | string | A unique ID for the request, which should be propagated to downstream systems. It should have a low probability of collision within a service in a temporal window measured in days. | |
```

Comment:
I find it strange to qualify the probability by "days". No proxy can guarantee that for unrestricted amount of traffic per day :)